imports/voting/ui/particles/SignStatement.vue

```diff
@@ -53,6 +50,14 @@
         <button>Submit</button>
       </form>
     </div>
+    <h5>Share this link</h5>
+    <span>{{`http://voting.aepps.com/proposal/${proposalId}`}}</span>
+    <button
+      class="btn-copy"
+      v-clipboard="`http://voting.aepps.com/proposal/${proposalId}`"
+      @success="onCopied"
+      @error="onCopyError"
+    >Copy</button>
```

Comment:
It should be a part of `ProposalDetail` page because `SignStatement` is also used in `CreateProposalModal`.

Reply:
How possible, because it's below the `{{ statement }}` which is inside `SignStatement.vue`

Reply:
I guess we need a prop to tell the component when it should show the copy button or not.